debian/libeos-knowledge-content-0-0.install

```diff
@@ -0,0 +1,2 @@
+usr/lib/libeos-knowledge-content-0.so.*
+usr/lib/gio/modules/libeknvfs.so
```

Comment:
@xjuan I didn't see the vfs library getting pulled into any existing packages, so I'm adding it here. Which I think is the right place for it anyway.

One thing I wasn't sure about was the gio-querymodules call here https://github.com/endlessm/eos-knowledge-lib/blob/286d25d82d7f1e35b2c7d42be0c359cd047ba1a3/Makefile.am#L302. If that's necessary it probably also needs to go in an post-install debian hook. Could you investigate and tag on a commit here in necessary?